base_layer/core/src/transactions/transaction_components/memo_field.rs

```diff
@@ -315,21 +315,21 @@ impl MemoField {
     }
 
     pub fn new_u256(value: U256) -> MemoField {
-        // U256 PaymentId: 1 byte for tag + 32 bytes for U256 = 33 bytes total
+        // U256 Memo: 1 byte for tag + 32 bytes for U256 = 33 bytes total
         // This always fits within 256 bytes, no runtime validation needed
         MemoField {
             inner: InnerMemoField::U256(value),
         }
     }
 
-    /// Helper function to create a validated `PaymentId::Open` from user data and transaction type
+    /// Helper function to create a validated `InnerMemoField::Open` from user data and transaction type
```

Comment:
wrong type?
The inner type is for practical use only, and should not be exposed for the docs. The function creates a type Memo, not InnerMemoField